app/models/UserAnswers.scala

```diff
@@ -29,8 +29,33 @@ final case class UserAnswers(internalId: String,
                              data: JsObject = Json.obj(),
                              lastUpdated: Instant = Instant.now) {
 
-  def getList[A](path: JsPath)(implicit rds: Reads[A]): Seq[A] =
-    path.read[Seq[A]].reads(data).getOrElse(Seq.empty)
+  private[models] def itemKeys: Seq[String] = (data \\ "items").flatMap {
+    case JsObject(underlying) => underlying.keys.toSeq
+    case _ => Seq()
+  }.toSeq
+
+  private[models] def get[A](key: String)(implicit reads: Reads[A]): Seq[A] = {
```

Comment:
Feel like this should be named `getItem` given that it only accesses the `items` object.